src/components/EventManagement/EventAttendance/Statistics/EventStatistics.tsx

```
@@ -0,0 +1,607 @@
+                  styles.topRightCorner + ' px-1 border border-success w-30'
+                }
+              >
+                <p className="text-black">{t('trends')}</p>
+              </div>
+              <div
+                className={
+                  styles.paddingBottom2Rem +
+                  ' d-flex position-absolute bottom-1 end-50 translate-middle-y'
+                }
+                role="navigation"
+                aria-label={t('chartPageNavigation')}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<nav>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)